Unify the style of the TWC articles (#2178)
ppy/osu-wiki: wiki/Tournaments/TWC/2016/en.md

---
tags:

> - TWC2016

Reviewer: ```suggestion
- TWC 2016
- TWC2016
```

---
# osu!taiko World Cup 2016

![TWC 2016 logo](img/logo.png)

The **osu!taiko World Cup 2016** (***TWC 2016***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the sixth installment of the osu!taiko World Cup.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2016-02-09/2016-02-28 |
| Live Drawings | 2016-03-11 (14:00 UTC) |
| Group Stage | 2016-03-19/2016-03-20 |
| Round of 16 | 2016-03-26/2016-03-27 |
| Quarterfinals | 2016-04-02/2016-04-03 |
| Semifinals | 2016-04-09/2016-04-10 |
| Finals | 2016-04-16/2016-04-17 |
| Grand Finals | 2016-04-23/2016-04-24 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu!taiko Champion" user title |

Reviewer 2: ```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu!taiko Champion" user title for one year |
```

- TWC 2016
- TWC2016
---
# osu!taiko World Cup 2016

> ![TWC 2016 logo](img/logo.png)

Reviewer 2: ```suggestion
![TWC 2016 Logo](img/logo.png)
```

- TWC 2016
- TWC2016
---
# osu!taiko World Cup 2016

![TWC 2016 logo](img/logo.png)

> The **osu!taiko World Cup 2016** (***TWC 2016***) was a country-based osu!taiko tournament hosted by the [osu! staff](/wiki/People/The_Team). It was the 6th installment of the osu!taiko World Cup.

Reviewer 2: ```suggestion
The **osu!taiko World Cup 2016** (***TWC 2016***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the sixth installment of the osu!taiko World Cup.
```
